updating/updating-cluster-between-minor.adoc

```diff
@@ -38,4 +38,12 @@ include::modules/update-service-overview.adoc[leveloffset=+1]
 
 include::modules/understanding-upgrade-channels.adoc[leveloffset=+1]
 
+include::modules/update-using-custom-machine-config-pools-ipsu.adoc[leveloffset=+1]
+
+If you want to use the canary rollout update process, see xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools[Performing an in-service software update].
+
 include::modules/update-upgrading-web.adoc[leveloffset=+1]
+
+= Next steps
+
+If you are performing an in-service software update, you must xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-unpause_update-using-custom-machine-config-pools[unpause the machine config pools] and xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-mcp-remove_update-using-custom-machine-config-pools[remove the nodes you added to a custom MCP].
```

Comment:
1. s/an in-service software update/a canary rollout update/
2. s/[remove the nodes you added to a custom MCP]/[move the nodes from a custom MCP to the original machine config pool]/